MatrixSDK/Data/MXRoomSummary.m

```diff
@@ -552,7 +554,9 @@ - (void)setIsEncrypted:(BOOL)isEncrypted
     // This should never happen
     if (_isEncrypted && !isEncrypted)
     {
-        MXLogError(@"[MXRoomSummary] setIsEncrypted: Attempt to reset isEncrypted for room %@. Ignote it. Call stack: %@", self.roomId, [NSThread callStackSymbols]);
```

Comment:
I'm presuming we don't need to worry about call stacks now that these go to Sentry.

Reply:
Exactly, though it will be missing from rageshake files ... I think that is okay?

Reply:
Yep yep, makes sense to me 👍